src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCaseDefinition.java

```diff
@@ -31,7 +31,16 @@
             "legalRepresentativeEmailAddress", new TypeReference<String>(){}),
 
     NOTIFICATIONS_SENT(
-            "notificationsSent",  new TypeReference<List<IdValue<String>>>(){});
+            "notificationsSent",  new TypeReference<List<IdValue<String>>>(){}),
+
+    HEARING_DATE_TIME(
+        "listCaseHearingDate",  new TypeReference<String>(){}),
```

Comment:
Think your formatting is off here.  Did you get the IA formatter xml loaded into your project?

Reply:
Yes, I have the IA_Codestyle loaded in my IDE. If I run the 'reformat code' on this particular file, it changes the formatting significantly (not sure why). I've adjusted the above to be in line with the other enums in the meantime.